CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 * Added methods `vapp.UpdateNetworkFirewallRules`, `vapp.UpdateNetworkFirewallRulesAsync`, `vapp.GetVappNetworkById`, `vapp.GetVappNetworkByName` and `vapp.GetVappNetworkByNameOrId` [#308](https://github.com/vmware/go-vcloud-director/pull/308)
 * Added methods `vapp.UpdateNetworkNatRulesAsync`, `vapp.UpdateNetworkNatRulesAsync`, `vapp.RemoveAllNetworkFirewallRules` and `vapp.RemoveAllNetworkNatRules` [#316](https://github.com/vmware/go-vcloud-director/pull/316)
 * Added methods `vapp.UpdateNetworkStaticRouting`, `vapp.UpdateNetworkStaticRoutingAsync` and `vapp.RemoveAllNetworkStaticRoutes` [#318](https://github.com/vmware/go-vcloud-director/pull/318)
+* Add OVF file upload support in UploadOvf function besides OVA. The input should be ovf file path, not ovf folder. It will check if input file is OVF file or not, if yes, skip some OVA specified steps, if not, keep old logic. [#323](https://github.com/vmware/go-vcloud-director/pull/323)
```

Comment:
Thanks! Suggesting few polishes:

```suggestion
* Add OVF file upload support in UploadOvf function besides OVA. The input should be OVF file path inside the OVF folder. It will check if input file is XML content type, if yes, skip some OVA steps (like unpacking), if not, keep the old logic. [#323](https://github.com/vmware/go-vcloud-director/pull/323)
```

Reply:
Thanks for the suggestion. I will commit in separate commit later.
Changes has been done following your comments.
Thanks.